system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
```

Comment:
What are we expecting to grow here to want this particular max-width? If it's the appropriate value, probably pull it out to a variable for the `.topsite-form`.

Reply:
I created 
```
$form-width: 300px;
$form-spacing: 32px;
```
And defined the width of the input + preview using those variables. The `.form-wrapper` has width 100% and can grow or shrink (the input width) when browser size is between the breakpoint (defined for the form) and full width.